doc/source/core_developer.md

```diff
@@ -80,6 +80,17 @@ When reviewing, focus on the following:
    tests not only execute the code, but explores corner cases.  It is tempting
    not to review tests, but please do so.
 
+5. **Licensing:** New contributions _must_ be compatible with  `scikit-image`'s 
+   license. Which licenses are compatible is a hard question to answer. If in
```

Comment:
Link to the `LICENSE.txt` file?

Reply:
Thanks, I'm on it but conda is taking its sweet time to install the dependencies to test it locally. :sweat_smile: